tests/suites/coslite/cl.sh

```diff
@@ -0,0 +1,71 @@
+# Ensure that COS Lite deploys successfully, the
+# relations are setup as expected and we can access the dashboards.
+run_deploy_coslite() {
+	echo
+
+	local model_name file overlay_path admin_passwd alertmanager_ip grafana_ip prometheus_ip
+	model_name="deploy-coslite"
+	file="${TEST_DIR}/${model_name}.log"
+	ensure "${model_name}" "${file}"
+
+	overlay_path="./tests/suites/coslite/overlay"
+	juju deploy cos-lite --trust --channel=edge --overlay "${overlay_path}/offers-overlay.yaml" --overlay "${overlay_path}/storage-small-overlay.yaml"
+
+	echo "Wait for all coslite bundle charms to be active"
+	wait_for "alertmanager" "$(active_condition "alertmanager" 0)" 1800
+	wait_for "grafana" "$(active_condition "grafana" 1)" 1800
+	wait_for "loki" "$(active_condition "loki" 2)" 1800
+	wait_for "prometheus" "$(active_condition "prometheus" 3)" 1800
+	wait_for "traefik" "$(active_condition "traefik" 4)" 1800
```

Comment:
Personally, I would still like one wait_for condition checking there are no non-active/idle applications

Reply:
And responding to:
> I addressed this by using active_condition

`active_condition` is exactly the same as checking for status being active as you did before, so the issue still exists. I'm still of the opinion `idle_condition` should be used

Reply:
I have had to use a custom check, for `idle_condition` does require passing in the unit index of a particular application which might make the test overly specific.